### PR TITLE
WTD AU Landing Page

### PR DESCRIPTION
**Pull request by swapnilogale**
a. Set up landing page
b. Create AU files

### CODE REVIEW

**Comment by swapnilogale** [comment]
@plaindocs I can't figure out what's failing on the RTD. Help please.

**Comment by plaindocs** [comment]
~@mxsasha would you mind taking a look at this?~ 

~The DataTemplateYAML class / import you added in https://github.com/writethedocs/www/pull/1255 seems to no longer work?~

~Possibly related to https://github.com/sphinx-contrib/datatemplates/pull/26/files ? Although that is pretty old.~

With packages up to date. I suspect I'm missing something really obvious here.

**Comment by plaindocs** [comment]
I suspect related to this https://sphinxcontribdatatemplates.readthedocs.io/en/latest/legacy.html but I don't know how to use the new syntax.

**Comment by plaindocs** [comment]
Well, that was a load of nonsense I wrote.

@swapnilogale you either need to add a ticket grant section to the config file, or you need to remove the following mentions of `grants`:

```
opportunity-grants.rst
20:**Grant applications are open until {{ grants.ends }}, Midnight {{ tz }}.**
59:	<iframe src="{{ grants.url }}?embedded=true" width="760" height="850" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
61:You can also view `the application form <{{ grants.url }}>`_ in its own page.

tickets.rst
108:You can apply until **{{ grants.ends }}, Midnight {{ tz }}** on `our website <https://www.writethedocs.org/conf/{{ shortcode }}/{{ year }}/opportunity-grants/>`_.
```

**Comment by plaindocs** [comment]
The YAML error seems to have been unrelated and possibly related to Python 3.6 on my local venv?

**Comment by swapnilogale** [comment]
@plaindocs more RTD errors? Seemed to be working fine a few days back.

**Comment by mxsasha** [comment]
> @plaindocs more RTD errors? Seemed to be working fine a few days back.

You haven't set any buttons in the "top" section in the config, like the sponsor button - which is enabled for the "bottom" section. I think that's the issue, and that the templates don't support that.

File: docs/conf/australia/2020/news/announcing-virtual-conf-dates.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+:template: {{year}}/generic.html
+
+.. post:: July 10, 2020
+   :tags: australia-2020, tickets, sponsors
+
+Announcing virtual conference dates
+===================================
+
+Write the Docs Australia 2020 will be a virtual conference. Plus this year, we are partnering with Write the Docs India for 2 days of Asia Pacific documentation goodness!
```
Comment (plaindocs) [inline docs/conf/australia/2020/news/announcing-virtual-conf-dates.rst]:
```suggestion
Write the Docs Australia 2020 will be a virtual conference. For the first time we are partnering with Write the Docs India for 2 days of Asia Pacific documentation goodness!
```

File: docs/conf/australia/2020/news/announcing-virtual-conf-dates.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+We decided to move the conference to December, to allow for enough time after the new Prague conference dates in October.
```
Comment (plaindocs) [inline docs/conf/australia/2020/news/announcing-virtual-conf-dates.rst]:
```suggestion
We're holding the conference in December, to allow for enough time after the new Prague conference dates in October.
```

But maybe we don't need to mention it at all?

File: docs/conf/australia/2020/news/cfp-open.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
+:template: {{year}}/generic.html
+
+.. post:: June 22, 2020
+   :tags: prague-2020, website, cfp, tickets
```
Comment (plaindocs) [inline docs/conf/australia/2020/news/cfp-open.rst]: Careful with these tags and dates!

File: docs/conf/australia/2020/ros.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+{{date.day_two.dotw}}, {{date.day_two.date}}
+-----------------------------------------
+
+{% if flaghasschedule %}
+{% with day_schedule=schedule.writing_day %}
+{% include "include/ros2020.rst" %}
+{% endwith %}
+{% endif %}
+
+Monday, October 19
```
Comment (plaindocs) [inline docs/conf/australia/2020/ros.rst]: !!

File: docs/conf/australia/2020/ros.rst [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+------------------
+
+{% if flaghasschedule %}
+{% with day_schedule=schedule.day1 %}
+{% include "include/ros2020.rst" %}
+{% endwith %}
+{% endif %}
+
+Tuesday, October 20
```
Comment (plaindocs) [inline docs/conf/australia/2020/ros.rst]: !!

**Review by plaindocs — COMMENTED** [review]
Couple of nitpicks

**Comment by swapnilogale** [comment]
@plaindocs - I am happy for this to be merged today. I'll keep working on the rest of the pages via a separate PR.

**Review by plaindocs — APPROVED** [review]
Looks good to me!

**Pull request merged**